src/Core/Security/Application/UseCase/LoginOpenIdSession/LoginOpenIdSession.php

```diff
@@ -87,18 +95,8 @@ public function __invoke(LoginOpenIdSessionRequest $request, LoginOpenIdSessionP
             }
             $this->provider->setConfiguration($openIdProviderConfiguration);
             $this->provider->authenticateOrFail($request->authorizationCode, $request->clientIp);
-            $user = $this->provider->getUser();
-            if ($user === null) {
-                if (!$this->provider->canCreateUser()) {
-                    throw new NotFoundException('User not found');
-                }
-                $this->info("User not found, start auto import");
-                $this->provider->createUser();
-                $user = $this->provider->getUser();
-                if ($user === null) {
-                    throw new NotFoundException('User not found');
-                }
-            }
+            $user = $this->findUserOrFail();
```

Comment:
Log the user's name

Reply:
user name is logged into the method getUser() call l.287

```
    /**
     * @inheritDoc
     */
    public function getUser(): ?ContactInterface
    {
        $this->info('Searching user : ' . $this->username);
        $user = $this->contactService->findByName($this->username);
        if ($user === null) {
            $user = $this->contactService->findByEmail($this->username);
        }

        return $user;
    }
 ```